### PR TITLE
fix: Fixed Button Color

### LINKED ISSUE #1435: Text Not Visibile in Button
**Describe the chore**
Text is not visible in Button when Internet Connectivity is not there.
Fix it.

Screenshot:
Screenshot_20190326-182231

**Would you like to work on the issue?**

- [x] Yes

### PR DESCRIPTION
**Pull request by somenath1435**
Fixes #1435 

Changes: Changed Button to Material Component Button.
Issue is fixed.

Screenshots for the change:
Before:
Screenshot_20190326-182231
After:
Screenshot_20190326-201241

### CODE REVIEW

**Comment by iamareebjamal** [comment]
We want a borderless button, like it was

**Comment by somenath1435** [comment]
@iamareebjamal 
New Looks:
Screenshot_20190326-212020

File: app/src/main/res/values/colors.xml [hunk 1]
```diff
@@ -16,4 +16,5 @@
     <color name="colorPrimaryVariant">#311B92</color>
     <color name="colorSecondary">#03A9F4</color>
     <color name="colorSecondaryVariant">#01579B</color>
+    <color name="white">#ffffff</color>
```
Comment (iamareebjamal) [inline app/src/main/res/values/colors.xml]: Remove

Comment (somenath1435) [reply app/src/main/res/values/colors.xml]: @iamareebjamal Done

**Pull request merged**